### PR TITLE
Update to webpack 4

### PR DESCRIPTION
**Pull request by kontrollanten**
Be aware that minification has been disabled for the OfflinePlugin according to https://github.com/NekR/offline-plugin/issues/351

### CODE REVIEW

**Comment by ajm113** [comment]
I've been using somewhat the same configuration. However, I think Webpack4 borks out for older browsers. At least that's what my tests show. -- Correct me if I'm missing something here...

File: webpack.config.babel.js [hunk 1]
```diff
@@ -34,6 +36,43 @@ module.exports = {
...
+						comparisons: true,
+						conditionals: true,
+						evaluate: true,
+						booleans: true,
+						loops: true,
+						unused: true,
+						hoist_funs: true,
+						if_return: true,
+						join_vars: true,
+						collapse_vars: true,
```
[the remark this answers is not in the document]
Comment (kontrollanten) [reply webpack.config.babel.js]: Well spotted! That's fixed now!

**Comment by tom-sherman** [comment]
@developit Looks like @kontrollanten has fixed the duplicate key, can this be merged now?

**Comment by developit** [comment]
@tom-sherman yes, we pretty much universally recommend preact-cli at this point since it includes a bunch of optimizations a boilerplate can never offer that result in smaller bundles, automatic code splitting, etc. There's a `@next` series of releases for preact-cli that are powered by Webpack 4.

**Pull request closed without merging**